zk/tests/conftest.py

```diff
@@ -87,3 +90,23 @@ def condition():
 
     with docker_run(compose_file, conditions=[condition]):
         yield get_instance
+
+
+@pytest.fixture
+def check():
+    return ZookeeperCheck(CHECK_NAME, {}, [VALID_CONFIG])
```

Comment:
(nit) I think you can just declare `zk_check = ZookeeperCheck(conftest.CHECK_NAME, {}, {})` in `test_metadata` like other tests.

Reply:
updated this here: https://github.com/DataDog/integrations-core/pull/4927/commits/717d15a88f36eef7ca4f39b81aee649de0bb5542